ui/keyboardMove/src/main.ts

```diff
@@ -28,6 +28,7 @@ export interface KeyboardMove {
   justSelected(): boolean;
   clock(): ClockController | undefined;
   draw(): void;
+  next?(): void;
```

Comment:
Essentially another nitpick, but I think it's a little cleaner if we don't leak the details of the type of root controller all the way into the KeyboardMove plugin, which this interface describes. By that I mean, making this method not optional, and instead handling the case where it is missing in this file. It would make it more consistent with the other methods. Admittedly in this case there's no functional difference.